src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -57,13 +55,13 @@ public class SqlScriptExecutor {
     @Autowired
     private DataSource pkgDataSource;
 
-    public int execute(SqlScript script) {
+    public int execute(SqlScript script, boolean throwInvalidObject) {
```

Comment:
to simplify code I propose to remove `throwInvalidObject` from `SqlScriptExecutor` and just call separate script after all executed scripts which will check invalid objects and throw exception if required

Reply:
put the check in a separate script and method.